### PR TITLE
[web-ui] #429 Fixes bug in website by not considering quoted string

### LINKED ISSUE #429: Website snippet search not working as intended to ..
I tried the snippet search against a java code block on this URL : http://stackoverflow.com/questions/2885173/how-to-create-a-file-and-write-to-a-file-in-java

It queries the following words:
printwriter, utf, the

where as it should have only queried printwriter

### PR DESCRIPTION
**Pull request by keerath**
fixes #429
closes #434

### CODE REVIEW

File: library/dev/js/modules/code-snippet-search.js [hunk 1]
```diff
@@ -9,9 +9,9 @@
     var searchTextArray = []
     // Replace all special characters with spaces.
     // Any character other than A-Z or a-z or 0-9 will be replaced by a blank
-    , clearedSelection = textSelected.replace(/[^a-zA-Z0-9_]/g, ' ').split(' ')
+    , clearedSelection = textSelected.replace(/(\"[^\"]*\")/g,'').replace(/[^a-zA-Z0-9_]/g, ' ').split(' ')
```
Comment (ScrapCodes) [inline library/dev/js/modules/code-snippet-search.js]: Can you try with regex `/(\"[^\"]*?\")/g` once, and see if it works too. I suppose the pattern should lazily applied.

**Comment by keerath** [comment]
@hashd PTAL..!!

**Comment by hashd** [comment]
PTAL @uidev547

**Comment by uidev547** [comment]
Verified with different patterns and working as expected.

**Pull request merged**